core/object/script_language.h

```diff
@@ -303,6 +303,7 @@ class ScriptLanguage {
 
 	void get_core_type_words(List<String> *p_core_type_words) const;
 	virtual void get_reserved_words(List<String> *p_words) const = 0;
+	virtual bool is_control_flow_keyword(String p_string) const { return false; };
```

Comment:
This method should be probably abstract, to be consistent with others (you wrote its definition for all classes anyways).